ci/zendesk_de-duplicator.yml

```diff
@@ -0,0 +1,56 @@
+resources:
+- name: zendesk-scripts
+  type: git
+  icon: github-circle
+  source:
+    branch: master
+    uri: "https://github.com/alphagov/zendesk-scripts.git"
+
+- name: hourly
+  type: time
+  icon: alarm
+  source:
+    location: {interval: 1h}
+
+- name: dedupe-repository
+  type: docker-image
+  source:
+    repository: "((readonly_private_ecr_repo_url))"
+    tag: concourse-dedupe-tickets-latest
+jobs:
+- name: build-zendesk-deduplicator
+  serial: true
+  plan:
+  - get: zendesk-scripts
+    trigger: true
+  - put: dedupe-repository
+    params:
+      build: zendesk-scripts
+
+- name: dedupe-tickets
+  serial: true
+  plan:
+  - get: hourly
+    trigger: true
+  - get: dedupe-repository
+    trigger: true
+    passed: [build-zendesk-deduplicator]
+  - task: run
+    image: dedupe-repository
+    config:
+      platform: linux
+      run:
+        args:
+        - -xce
+        - |
+          cd /usr/src/app
+          export ZENDESK_LOG_FILE=zendesk-dedupe-tickets.`date +%Y-%m-%d%H%M`
+          bundle exec ruby /usr/src/app/lib/zendesk-ticket-deduplicator.rb
+          aws s3 cp $ZENDESK_LOG_FILE s3://${S3_BUCKET_NAME}/
```

Comment:
I assume this log is being created and shipped to s3 because whomever wants to read it doesn't have concourse access but does have s3 access?

Reply:
Correct.